### PR TITLE
remove extra comma

### PR DESCRIPTION
**Pull request by joejulian**
There should be no comma in the certificate subj. Commas are interpreted as part of the string, ie `system:nodes,` instead of `system:nodes`.

### CODE REVIEW

File: assets/generate_apiserver.sh [hunk 1]
```diff
@@ -7,7 +7,6 @@ mkdir -p /output/etcd/ssl
 
 # Variables are:
 # SERVICE_DNS_NAME - defaults to localhost
-# SHORT_HOST_NAME - defaults to localhost
```
Comment (philoserf) [inline assets/generate_apiserver.sh]: Any comment on this apparently unrelated change?

**Comment by joejulian** [comment]
You got me... that was supposed to be in $last_pr but I missed it.
